box_it/index.js

```diff
@@ -0,0 +1,87 @@
+//Test data for boxit
+myArray = ["dsgffgddfg", "iggdfgdfs", "adfgddffg", "ByteStream", "test"];
+
+
+function maxStrLength(arrStr) {
+
+    let mstrlgt = 0;
+    for (str in arrStr) {
+        if (arrStr[str].length > mstrlgt) {
+            mstrlgt = arrStr[str].length;
+
+        }
+
+    }
+    return mstrlgt;
+}
+
+//console.log(maxStrLength(myArray));
+
+
+function boxIt(arrStr) {
+    console.log(drawTopBorder(maxStrLength(arrStr)) + '\n');
+
+    for (item in arrStr) {
+
+        let spaces = addSpaces(arrStr[item], (maxStrLength(arrStr)));
+      if (arrStr.indexOf(item) !== 0){
+        console.log(drawMiddleBorder(maxStrLength(arrStr)) + '\n');
+      }
+        console.log(drawBarsAround(spaces));
+       
+    }
+    console.log(drawBottomBorder(maxStrLength(arrStr)) + '\n');
+
+
+}
+function drawLine(num) {
+    let line = '━';
```

Comment:
If a variable does not need to be reassigned it's best practice to declare them with `const`.
This way you prevent accidentally re-assigning the variable's value

As a rule of thumb always use `const` first and if the variable needs to change then use `let`